source/faq/faq.rst

```diff
@@ -311,6 +311,17 @@ It is important to point out that dojot contains no e-mail server. It will gener
 and send them to the specified e-mail server.
 
 
+What about a HTTP POST request, how can I send it?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is almost the same process as sending an e-mail. 
```

Comment:
Wouldn't it be nice to have a textual description as well?

Reply:
We tend to avoid explaining GUI actions with text - they are extremely sensitive to changes and definetly not so easy to write down. We would end up with a buch of paragraphs and a lot of screenshots. It is much more informative to have animated GIFs for that.

Reply:
Ok Dok